src/ui/page/search/index.html

```diff
@@ -33,6 +24,11 @@ <h1 class="heading heading-large heading--flush">
             <div class="column-two-thirds">
                 <div class="resources">
                     {#if searchResults.combinedResults && searchResults.combinedResults.length}
+                     {#if searchResults}
+                    <p class="lede search-lede">
+                       { $req.__n('%s result', searchResults.totalResults) } found
+                    </p>
+                {/if}
```

Comment:
Indentation needs sorting out here, might be a tabs and spaces rendering differently thing